src/LinearFractional.jl

```diff
@@ -369,10 +394,18 @@ end
 function MOI.ScalarAffineFunction(a::GenericAffExpr{Float64,LinearFractionalVariableRef})
     _assert_isfinite(a)
     terms = MOI.ScalarAffineTerm{Float64}[MOI.ScalarAffineTerm(t[1],
-                                                               index(t[2].vref))
+                                                               index(t[2]))
```

Comment:
Can you explain this change?  Why is `.vref` no longer needed here?

Reply:
Thanks for taking a look! This `vref` is no longer needed because I added `index` to the list of functions being forwarded at the beginning of the file (that is,  `@forward LinearFractionalVariableRef.vref`). I am happy to revert this (since it is not directly related to the purpose of this PR---apologies).